crates/bevy_render/src/mesh/primitives/dim2.rs

```diff
@@ -886,14 +887,14 @@ impl Capsule2dMeshBuilder {
 impl MeshBuilder for Capsule2dMeshBuilder {
     fn build(&self) -> Mesh {
         // The resolution is the number of vertices for one semicircle
-        let resolution = self.resolution as u32;
-        let vertex_count = 2 * self.resolution;
+        let resolution = self.resolution;
+        let vertex_count = 2 * resolution;
```

Comment:
Wouldn't doing

```rust
let vertex_count = 2 * resolution as usize;
```

let us trim `as usize` in lines 895 - 897?

Reply:
Yes, but we would get the opposite back in later lines, in more unfortunate places :thinking:

Reply:
Gotcha 👌